"make release-manifests" no longer works

To run `clusterctl config cluster` locally, one must generate the `infrastructure-components.yaml` file which is done by `make release-manifests`, but this command currently prints the following error:
```
$ make release-manifests
release version v0.0.0
KUSTOMIZE_ENABLE_ALPHA_COMMANDS=true /tmp/test/cluster-api-provider-tink/hack/tools/bin/kustomize-v3.8.8 cfg set config image-tag v0.0.0
Error: unable to find "Krmfile" in package "config"
make: *** [Makefile:364: release-version] Error 1
```
CC @cpanato

--- a/Makefile
+++ b/Makefile
@@ -162,7 +162,7 @@ CLUSTERCTL_TEMPLATE ?= templates/clusterctl-template.yaml
 CLUSTER_TEMPLATE ?= templates/cluster-template.yaml
 
 # Binaries.
-KUSTOMIZE_VER := v3.8.8
+KUSTOMIZE_VER := v3.9.2
 KUSTOMIZE_BIN := kustomize
 KUSTOMIZE := $(TOOLS_BIN_DIR)/$(KUSTOMIZE_BIN)-$(KUSTOMIZE_VER)
 

--- a/config/Krmfile
+++ b/config/Krmfile
@@ -1,0 +1,11 @@
+apiVersion: config.k8s.io/v1alpha1
+kind: Krmfile
+openAPI:
+  definitions:
+    io.k8s.cli.setters.image-tag:
+      type: string
+      x-k8s-cli:
+        setter:
+          name: image-tag
+          value: v0.0.0-dirty
+          isSet: true

